apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/ProfileDetails/ProfileDetails.tsx

```diff
@@ -0,0 +1,141 @@
+import * as React from "react";
+import { useIntl } from "react-intl";
+
+import { User } from "@gc-digital-talent/graphql";
+import {
+  commonMessages,
+  getCitizenshipStatusesAdmin,
+  getLanguage,
+  getProvinceOrTerritory,
+} from "@gc-digital-talent/i18n";
+import { Link, Well } from "@gc-digital-talent/ui";
+
+interface ProfileDetailsProps {
+  user: User;
+}
+
+const ProfileDetails = ({ user }: ProfileDetailsProps) => {
+  const intl = useIntl();
+
+  const location = (
+    <p>
+      {user.currentCity || intl.formatMessage(commonMessages.notFound)},{" "}
+      {intl.formatMessage(
+        user.currentProvince
+          ? getProvinceOrTerritory(user.currentProvince)
+          : commonMessages.notFound,
+      )}
+    </p>
+  );
+
+  const telephone = (
+    <p>
+      {user.telephone ? (
+        <Link
+          external
+          href={`tel:${user.telephone}`}
+          aria-label={user.telephone.replace(/.{1}/g, "$& ")}
+        >
+          {user.telephone}
+        </Link>
+      ) : (
+        ""
+      )}
+    </p>
+  );
+
+  const email = (
+    <Link external href={`mailto:${user.email}`}>
+      {user.email}
+    </Link>
+  );
+
+  const citizenship = (
+    <p>
+      {intl.formatMessage(
+        user.citizenship
+          ? getCitizenshipStatusesAdmin(user.citizenship)
+          : commonMessages.notFound,
+      )}
+    </p>
+  );
+
+  const preferredLang = (
+    <p>
+      {intl.formatMessage({
+        defaultMessage: "General Communication:",
+        id: "gU7Nup",
+        description: "Label for preferred language in profile details box.",
+      })}{" "}
+      {intl.formatMessage(
+        user.preferredLang
+          ? getLanguage(user.preferredLang)
+          : commonMessages.notFound,
+      )}
+    </p>
+  );
```

Comment:
This might seem heav handed but for all of these, we have a dividing colon message so we don't need to add the extra space and ensure the translations are consistent since French uses different spacing.

https://github.com/GCTC-NTGC/gc-digital-talent/blob/f206cfe084b11e0df48dd80be3364656e4617ed3/packages/i18n/src/messages/commonMessages.ts#L98-L102

Reply:
Fixed by [00e51c1](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8853/commits/00e51c12c07223f004bf18b2062f4bd941743b58)